src/rebar_utils.erl

```diff
@@ -158,11 +158,21 @@ sh(Command0, Options0) ->
     ?DEBUG("Port Cmd: ~s\nPort Opts: ~p\n", [Command, PortSettings]),
     Port = open_port({spawn, Command}, PortSettings),
 
-    case sh_loop(Port, OutputHandler, []) of
-        {ok, _Output} = Ok ->
-            Ok;
-        {error, {_Rc, _Output}=Err} ->
-            ErrorHandler(Command, Err)
+    case os:type() of
+        {win32, _} ->
```

Comment:
Different is on os.

*nix system when doing copy returns 0 if it is ok. Robocopy returns values from 0 - 8 and 16 if nothing was copies (This can be an ok or sometimes considered a error)

so using the sh_loop will fail when doing copy. I thought it would be better with a own sh_loop for windows handling exit codes than in functions using robocopy do.

Res = rebar_utils:sh("robocopy blah blah"...) 
case Res of
 {error, {1, []}} -> ...

I thought it would be better to handle the codes that is ok and fail if we get something else.

If Robocopy copies files into a directory where there is already files in it it can give a exit code 3. and this will fail if only is looking for a 1.